apps/api_web/lib/api_web/params.ex

```diff
@@ -203,6 +203,13 @@ defmodule ApiWeb.Params do
   def route_types(%{"route_type" => route_types}), do: integer_values(route_types)
   def route_types(_), do: []
 
+  @doc """
+  Parse is_canonical filter param into integer
+  """
+  def is_canonical("true"), do: 1
+  def is_canonical("false"), do: 2
+  def is_canonical(_), do: nil
```

Comment:
question: Shouldn't a route with no canonical patterns be included by `filter[is_canonical]=false`? Looking at the RFC, `is_canonical = 0` indicates that a route has no canonical pattern defined. I see below in `RoutePattern.filter_by/1` you're filtering routes that don't match this number parsed out of the filter param, but it's not clear to me that this is the correct behavior, because it means that you'll never see routes with no canonical pattern defined when using this filter. Is that intended?

Reply:
agreed, will update based on the is true or everything else vs specifically false.